modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -2765,8 +2771,11 @@ public void changedLocation(float x, float y) {
 
         @Override
         public void changedSize(float w, float h) {
-            if (w != Scene.this.getWidth()) Scene.this.setWidth(w);
-            if (h != Scene.this.getHeight()) Scene.this.setHeight(h);
+            boolean widthChanged = w != Scene.this.getWidth();
+            boolean heightChanged = h != Scene.this.getHeight();
+            if (widthChanged) Scene.this.setWidth(w);
```

Comment:
it might be my personal preference, but I would rather see one statement per line.  use case: stepping through in the debugger.  can we add { }'s please?